pkg/game/persistence.go

```diff
@@ -34,3 +34,51 @@ func dbCreateGameState(c *pgxpool.Pool, gameid int) error {
 	return err
 
 }
+
+func dbUpdateResource(c *pgxpool.Pool, gameID int, resource string) error {
```

Comment:
The function name seems to suggest that this updates a resource but the logic only seems capable of adding a resource. Could we make this more generic so it can add or remove a resource?